NewHorizons/Builder/Body/GeometryBuilder.cs

```diff
@@ -12,8 +12,16 @@ public static GameObject Make(GameObject planetGO, Sector sector, float groundSc
             groundGO.transform.parent = sector?.transform ?? planetGO.transform;
             groundGO.transform.localScale = new Vector3(groundScale, groundScale, groundScale);
             groundGO.transform.position = planetGO.transform.position;
-            groundGO.GetComponent<MeshFilter>().mesh = SearchUtilities.Find("CloudsTopLayer_GD").GetComponent<MeshFilter>().mesh;
```

Comment:
Is the top layer not literally just a sphere? This is original marshmallow code so I'm not sure. If it is we could simplify that.

Reply:
It is a sphere but the giant's deep top layer is a rounder sphere than the primitive unity sphere.